protocol.md

```diff
@@ -211,6 +211,10 @@ bytes contained in the message at the given offset specified by the
 `Upload-Offset` header. All `PATCH` requests MUST use
 `Content-Type: application/offset+octet-stream`.
 
+If the Request-URI does not point to an existing resource, the server MAY create 
+a new resource if the request fullfills the preconditions of a corresponding
+`POST` request.
+
```

Comment:
If _precondition_ term is used here in the same sense as in HTTP protocol [specification](https://tools.ietf.org/html/rfc7231#section-5.2), a set of validators for a resource state should be specified. Otherwise the term should be expanded in more clear way.